### DISCUSSION TITLE
Does DB4S have plans to support Qt6?

### DISCUSSION THREAD

**Discussion by yzhgu** [sqlitebrowser/sqlitebrowser#3606] [General]
att

**Comment by justinclift** [comment]
At some point it's likely to happen yes.  We've not really started putting any effort into that though.

**Reply by HubKing** [reply]
It has been very problematic for me, because QT5 does not support Wayland. Under Gnome, it looked blurry. Under Plasma, X11 apps don't get scaled by default, so I used QT_SCALE_FACTOR, but it causes menu mouse position to be wrong. Also, it does not seem to follow the system style, making disabled text illegible.

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/10900489/a31695f9-8221-4ef0-989f-7960869ce436)

**Reply by justinclift** [reply]
Thanks @HubKing, that's good info. :smile:

**Reply by HubKing** [reply]
Well, I just discovered that it has its own theme/font settings. It had "Follow the desktop style" and "Dark", and I switched to Dark, set the font size to 16, and disabled QT SCALING. I don't like this blue-greenish colour scheme, but I think it's more useable now. Are there more themes than this "Dark"?

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/10900489/7eb8e1a6-9225-4b14-8900-a67dbee4eb45)

**Reply by justinclift** [reply]
Nah, it's just Light and Dark.  So far anyway. :smile:

**Comment by WhyNotHugo** [comment]
See https://github.com/sqlitebrowser/sqlitebrowser/pull/3711
